.spectral.yaml

```diff
@@ -1,254 +0,0 @@
-# https://meta.stoplight.io/docs/spectral/cb95cf0d26b83-core-functions
-extends:
```

Comment:
isn't the spectral in mms still pointing to this file?

Reply:
no https://github.com/10gen/mms/blob/48399666b300669ed5717040045e04710a6105f4/scripts/evergreen/openapi/.spectral.yaml#L4

Reply:
Thank for checking! another question: is the GH action release updated already to use the new file?

Reply:
I'm thinking yes since I updated in the previous PR and Spectral Lint Commited OpenAPI Spec is not failing

Reply:
yes, you did  https://github.com/mongodb/openapi/blob/0290379ff307d66aefb30f87209c05495514df1b/.github/workflows/required-spec-validations.yml#L36 👍